### PR TITLE
Fixed typo in output.md

### PR DESCRIPTION
**Pull request by nadirlaskar**
Fixed a typo in formatting, at line `731`.
`require`d -> `required`

### CODE REVIEW

File: src/content/configuration/output.md [hunk 1]
```diff
@@ -728,7 +728,7 @@ There is no need to change it.
 
 `boolean`
 
-Tell webpack to remove a module from the module instance cache (`require.cache`) if it throws an exception when it is `require`d.
+Tell webpack to remove a module from the module instance cache (`require.cache`) if it throws an exception when it is `required`.
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md]: isnt `require` specifically wrap to imply a command here? or is there a `required` directive?

**Comment by montogeek** [comment]
@nadirlaskar Friendly ping

**Comment by EugeneHlushko** [comment]
This isnt a valid addition, author hasn't come back with reasoning behind the PR, closing for now

File: src/content/configuration/output.md (hunk 1 above)
Comment (nadirlaskar) [reply src/content/configuration/output.md:843]: Sorry, about that I think my understanding was not correct.
We can close the PR.

**Pull request closed without merging**